code/game/atoms/_atom.dm

```diff
@@ -123,6 +123,9 @@
 	var/list/managed_vis_overlays
 	///The list of alternate appearances for this atom
 	var/list/alternate_appearances
+	///var containing our storage, see atom/proc/create_storage()
+	var/datum/storage/atom_storage
```

Comment:
var/datum/storage/storage better IMO, as we discussed

Reply:
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/66163761/8762bc17-e697-47b2-976c-7e727a82bf46)
My issue with this is it would make it basically impossible to find any implementation of our storage var.

Is it acceptable if I rename it to storage_datum or datum_storage instead as a var name?
If you really want me to rename it to storage/storage I will, but it would make it incredibly difficult to work with in the future for anyone who wants to make any changes to it.

Reply:
Good argument, I forgot we have shitty LSP.

Ok for storage_datum